src/reporting/claim-reporting-fees.js

```diff
@@ -221,33 +253,37 @@ function redeemContractFees(p, payload, successfulTransactions, failedTransactio
 }
 
 /**
- * TODO: Add updated JSDoc info for input/returned values.
- *
  * Claims all reporting fees for a user as follows:
  *
- * If the forked market is finalized:
- *   Call `Market.migrateThroughOneFork` for all non-forked, non-finalized non-migrated markets in the same universe as the forked market.
- * Else:
- *   Call `Market.disavowCrowdsourcers` for all non-forked, non-finalized, non-disavowed markets in the same universe as the forked market.
+ * If a forked market exists in the current universe:
+ *   For all non-finalized markets in current universe where the user has unclaimed fees in the reporting participants:
+ *     Call `Market.disavowCrowdsourcers`
  *
- * Once the above transactions are finished:
- *   Call `FeeWindow.redeem` on all specified FeeWindows
- *   For each DisputeCrowdsourcer the user unredeemed staked in:
- *     If its market is forked and `DisputeCrowdsourcer.fork` has not been called:
- *       Call `DisputeCrowdsourcer.forkAndRedeem`
- *     Else:
- *       Call `DisputeCrowdsourcer.redeem`
- *   For each InitialReporter the user has unredeemed stake in:
- *     If its market is forked and `InitialReporter.fork` has not been called:
- *       Call `InitialReporter.forkAndRedeem`
+ * Once the above has been completed:
+ *   Call `FeeWindow.redeem` on all fee windows in the current universe where the user has unclaimed participation tokens
+ *   For each market participant of the forked market in which the user has unclaimed fees:
```

Comment:
market participant == reporting participant?

Reply:
Yeah, that's a typo. Will fix.